docs/platform/webcontainers/turbo-package-manager.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: &title Turbo Package Manager
-description: &description WebContainers-based projects use Turbo as package manager. Turbo is our custom npm client and it works similarly to `npm` and `yarn`. 
+description: &description Turbo has been replaced with native npm support, alongside native pnpm and yarn v1. 
```

Comment:
Nit: I would phrase this differently for users that don't know about it. Maybe something like:

```
Turbo was our custom npm client until very recently when native npm was added to WebContainer. It is available until January 2024.
```

Reply:
done 👍